ipv8/test/messaging/anonymization/mock.py

```diff
@@ -0,0 +1,21 @@
+from collections import defaultdict
+
+# Map of info_hash -> peer list
+global_dht_services = defaultdict(list)
+
+
+class MockDHTProvider:
```

Comment:
**nit:** probably this variable should be placed inside 'MockDHTProvider' namespace
```suggestion

class MockDHTProvider:
    global_dht_services = defaultdict(list)
```